doc/rbd/rbd-snapshot.rst

```diff
@@ -42,18 +45,22 @@ Cephx Notes
 When `cephx`_ authentication is enabled (it is by default), you must specify a
 user name or ID and a path to the keyring containing the corresponding key. See
 :ref:`User Management <user-management>` for details. You may also set the
-``CEPH_ARGS`` environment variable to avoid re-entry of these parameters. ::
+``CEPH_ARGS`` environment variable to avoid re-entry of these parameters:
+
+.. prompt:: bash $
 
 	rbd --id {user-ID} --keyring=/path/to/secret [commands]
 	rbd --name {username} --keyring=/path/to/secret [commands]
 
-For example:: 
+For example:
+
+.. prompt:: bash $
 
 	rbd --id admin --keyring=/etc/ceph/ceph.keyring [commands]
 	rbd --name client.admin --keyring=/etc/ceph/ceph.keyring [commands]
 
-.. tip:: Add the user and secret to the ``CEPH_ARGS`` environment 
-   variable so that you don't need to enter them each time.
+.. tip:: Add the user and secret to the ``CEPH_ARGS`` environment variable to
+   avoid re-entry of these parameters.
```

Comment:
This tip pretty much duplicates the last sentence of the previous paragraph.  I'd drop it and keep the tip as the tip gets nicely highlighted.

Reply:
Done. I removed the sentence and kept the tip.